app/src/main/res/layout/app_language_activity.xml

```diff
@@ -36,7 +36,7 @@
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/reading_list_app_bar_layout">
+      app:layout_constraintTop_toBottomOf="@id/reading_list_app_bar_layout" >
```

Comment:
We should drop this space, right?

Reply:
Yeah I can do that--it was originally in the file so it was copied over during reversion.

Reply:
(Marking as resolved since this is a trivial change and you've already approved @seanlip).